core/src/main/scala/org/apache/spark/rdd/PairRDDFunctions.scala

```diff
@@ -1014,7 +1015,7 @@ class PairRDDFunctions[K, V](self: RDD[(K, V)])
    * Output the RDD to any Hadoop-supported file system, using a Hadoop `OutputFormat` class
    * supporting the key and value types K and V in this RDD.
    *
-   * Note that, we should make sure our tasks are idempotent when speculation is enabled, i.e. do
+   * @note We should make sure our tasks are idempotent when speculation is enabled, i.e. do
```

Comment:
Ah, I left and removed a useless comment. Yes, it seems it goes together.

<img width="972" alt="2016-11-18 1 41 26" src="https://cloud.githubusercontent.com/assets/6477701/20418861/d2527450-ad94-11e6-9cac-fded62830d90.png">
<img width="870" alt="2016-11-18 1 42 02" src="https://cloud.githubusercontent.com/assets/6477701/20418862/d256caaa-ad94-11e6-9b4c-c0add3555d4a.png">